frontend/test/lit/test_quantum_control.py

```diff
@@ -51,6 +53,19 @@ def __init__(self, shots=None, wires=None):
             custom_capabilities.native_ops.update(additions)
             self.qjit_capabilities = custom_capabilities
 
+        @staticmethod
```

Comment:
You removed lightning, that should not be affected non? https://github.com/PennyLaneAI/catalyst/pull/1227/files#diff-9b581cb697791c9f6d88df9b2e7100d928f97106f55e1b65b0b4053c69275457R129

Reply:
If a `get_c_interface` method is not provided for a custom device, the following error raises:
```
File "/home/raul/repo/catalyst/frontend/catalyst/device/qjit_device.py", line 182, in extract_backend_info
    raise CompileError(f"The {dname} device does not provide C interface for compilation.")
catalyst.utils.exceptions.CompileError: The lightning.qubit device does not provide C interface for compilation.
```